qa-testapp/src/main/java/com/mapbox/navigation/qa/view/adapters/ActivitiesListAdapterSupport.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.navigation.qa.view.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.mapbox.navigation.qa.R
+import com.mapbox.navigation.qa.domain.model.TestActivityDescription
+import kotlinx.android.synthetic.main.test_activity_item_layout.view.*
+
+object ActivitiesListAdapterSupport {
+
+    val activitiesListOnBindViewHolderFun: (
+        holder: ActivityViewHolder,
+        position: Int,
```

Comment:
Why pass position if it's unused?

Reply:
I copy/pasted from a library I use in my personal work.  Since it's a general purpose RecycleView adapter it includes the position because sometimes there's a need for it.  It may be useful in the future and it doesn't get in the way so I suggest keeping it unless you feel strongly it should be removed.

Reply:
This is meant as a general purpose abstraction the position parameter doesn't get in the way of anything. I think we should keep it. It's easily ignored.

Reply:
:+1: